source/renderer/app/stores/HardwareWalletsStore.ts

```diff
@@ -1773,6 +1773,13 @@ export default class HardwareWalletsStore extends Store {
             }
           );
         }
+      } else if (error.code === 'Device_InvalidState') {
```

Comment:
@marcin-mazurek we need to check if there is just one message for this error code or we can get multiple different messages under the same invalid state code. 
Also, in this case, we should let the user know that the device must be reconnected if the wrong passphrase is entered. 
p.s. When you are exporting a public key for the first time there is no wrong passphrases! But, if you want to e.g. sign the transaction under the same session you will be asked to enter a passphrase ONLY on the first attempt. After that, your passphrase is stored in the device session. Those are really important information you should keep in mind while you are working on HW's. And, Ledger work in a different way, where an alternate PIN is introduced.